ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -44,9 +49,29 @@ public PatientVisitsDashboardPage(Page parent, WebElement waitForStaleness) {
     public String getPageUrl() {
         return "coreapps/patientdashboard/patientDashboard.page";
     }
+    
+    public PatientCaptureVitalsPage goToPatientCaptureVitalsPage() {
+        waiter.until(ExpectedConditions.presenceOfElementLocated(CAPTURE_VITALS));
+        clickOnLast(CAPTURE_VITALS);
```

Comment:
@kdaud  she have added it here, @Irenyak1  is this how the workflow is supposed to flow ?

Reply:
Thanks @sherrif10 for pointing this out. It actually makes the return type to be true for the method call.

One thing I would recommend here is to rather use clickOn() when clicking on the page elements.

Reply:
> Thanks @sherrif10 for pointing this out. It actually makes the return type to be true for the method call.
> 
> @Irenyak1 one thing I would recommend here is to rather use `clickOn()` when clicking on the page element.

@kdaud  I get it. Are there any other comments I should address before I make another push?

Reply:
> Are there any other comments I should address before I make another push?

If I was @Irenyak1, I would not always wait reviewers to put all their comments on the PR then I address them at once. Why? 
Reviewing a PR is a process and can take more time when the PR author is not addressing the pending reviews waiting for all the reviews at once. 

I learnt addressing any review on my PR immediately and push the requested changes. Then wait for any other review(s) and if any I immediately address them as well. This saves my time when working on a PR especially when reviewers are giving quick feedback. 

I learnt this logic from @dkayiwa and am now a consultant to my self.

Reply:
@kdaud of course, I can not sit and wait. I attended to all the reviews long time ago. Just waiting for others if any, as I do something else. No time to Waste!